Tests/Data/ThermoHydroMechanics/Linear/Point_injection/square_1e2.prj

```diff
@@ -293,6 +293,7 @@
                 <precon_type>ILUT</precon_type>
                 <max_iteration_step>10000</max_iteration_step>
                 <error_tolerance>1e-8</error_tolerance>
+                <scaling>1</scaling>
```

Comment:
Not used by BiCGSTAB. Only for SparseLU.

Reply:
This is also what I thought, but it makes a huge difference for convergence and run time:
with scaling:
info: === Time stepping at step #1 and time 5000 with step size 5000
info: Calculate non-equilibrium initial residuum.
info: [time] Assembly took 0.038145 s.
info: [time] Applying Dirichlet BCs took 0.00581064 s.
info: ------------------------------------------------------------------
info: *** Eigen solver computation
info: -> scale
info: -> solve with BiCGSTAB (precon ILUT)
info: 	 iteration: 1/10000
info: 	 residual: 6.772733e-09

info: ------------------------------------------------------------------
info: [time] Linear solver took 0.163631 s.
without: 
info: === Time stepping at step #1 and time 5000 with step size 5000
info: Calculate non-equilibrium initial residuum.
info: [time] Assembly took 0.037899 s.
info: [time] Applying Dirichlet BCs took 0.0050468 s.
info: ------------------------------------------------------------------
info: *** Eigen solver computation
info: -> solve with BiCGSTAB (precon ILUT)
info: 	 iteration: 369/10000
info: 	 residual: 6.246455e-09

info: ------------------------------------------------------------------
info: [time] Linear solver took 1.74722 s.